advanced-usage/index.md

```diff
@@ -92,10 +130,9 @@ Multiple simultaneous tabs from the same browser instance will be supported in f
 - New widgets can be added via the bottom section
     - Clicking on a regular widget adds it to the view, after which it can be positioned and resized as desired
     - Mini widgets have fixed sizes, but can be dragged and dropped into the desired location in the header/footer
-      bars or in a [mini widget container](#mini-widget-container)
-- Some regular widgets can be configured, by clicking the three dots in the top right corner of the widget
-- Mini widgets that support configuration can be clicked on anywhere on the widget to open the configuration options
-    - This is currently only possible during operation, *not* while in edit mode
+      bars or in a custom [mini widget bar](#mini-widget-bar)
+    - The selector in the bottom left can be used to choose between editing regular or mini widgets
+- Some widgets can be configured, by clicking the cog settings icon in the "Current widgets" list
```

Comment:
"If you click in the cog icon and nothing appears, it means this widget cannot be configured."

We should mention it until we fix it.